developers/weaviate/manage-data/import.mdx

```diff
@@ -90,6 +90,13 @@ The following example adds objects to the `MyCollection` collection.
   </TabItem>
 </Tabs>
 
+### Error handling
+
+During a batch import, any failed objects or references will be stored and can be obtained through `batch.failed_objects` and `batch.failed_references`. 
+Additionally, a running count of failed objects and references is maintained and can be accessed through `batch.number_errors` within the context manager.
```

Comment:
Maybe we could make a suggestion, that they could use `batch.number_errors` to interrupt the loop and if they get too many errors.